### PR TITLE
updated ping source with kn base64 example

### LINKED ISSUE #5058: [CLI] Document using kn to create a PingSource
Update the docs here https://knative.dev/docs/eventing/sources/ping-source/ to include `kn` examples

### PR DESCRIPTION
**Pull request by amaanq**
Fixes #5058

- Added an example of using kn to create a ping source with base64 binary data

### CODE REVIEW

**Comment by amaanq** [comment]
Just signed the CLA authorization, not sure how to redo the checks

**Review by aslom — APPROVED** [review]
/lgtm

File: docs/eventing/sources/ping-source/README.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ during the verification step in this procedure.
 
 ## Create a PingSource object
 
-1. Optional: Create a namespace for your PingSource by running the command:
+1. Optional: Create a namespace for your PingSource by running this command:
```
Comment (abrennan89) [inline docs/eventing/sources/ping-source/README.md:26]: Should be left as "the" > "this", per the style guide: https://github.com/knative/docs/blob/main/contribute-to-docs/style-guide/documenting-code.md#documenting-commands

**Review by abrennan89 — APPROVED** [review]
/lgtm
/unhold

**Comment by amaanq** [comment]
I believe I fixed the issues described, let me know!

**Pull request merged**